README.md

```diff
@@ -1 +1,61 @@
-# space-travelers-hub2.0
\ No newline at end of file
+# Space-Travelers-Hub
+In this task, we will be working with the real live data from the SpaceX API. Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+# Screenshots
+
+## Rocket and Home Page:
+![Screenshot (44)](https://user-images.githubusercontent.com/71400898/151654290-050ff747-f8d7-4594-a30b-df1887f84b19.png)
+
+## Missions Page:
+![Screenshot (45)](https://user-images.githubusercontent.com/71400898/151654297-7cc942dc-34a2-491b-8fb2-3472c56ce480.png)
+
+## Profile Page:
+![Screenshot (46)](https://user-images.githubusercontent.com/71400898/151654304-f9c2dc0b-92f4-4da2-acb3-ab1686d69b3e.png)
+
+
+## Built With
+- React-Redux
+- Javascript
+- React Bootstrap
+- 
+### Technologies used
+- Github
+- Gitflow
+- Visual Studio Code
+- 
+## Live Demo
+[Link]()
```

Comment:
- [ ] Just one more to go 🔢 . Let's deploy this great app to Heroku or [Netlify](https://www.netlify.com/blog/2016/07/22/deploy-react-apps-in-less-than-30-seconds/) or Github so as to have a live version of this great project in the README for visitors to easily get to see your application.